tracker/tracker_object.rst

```diff
@@ -569,6 +569,10 @@ Anonymization
 Advanced Usage
 --------------
 
+.. function:: ping()
+
+    Ping method may be used to update session dimensions during page view without attaching them to actions.
```

Comment:
Don't we want to list what dimensions are updated? What are the most typical use cases?

Reply:
I think it may be described too briefly. It should be
```suggestion
    Ping method may be used to update session custom dimensions during page view without attaching them to actions.
```

Reply:
That better but still not entirely true, because it updates not only custom dimensions.
I'm not saying my version is perfect but here's what I've commited lately: https://github.com/PiwikPRO/PPMS-PublicDocs/blob/15.2/tracker/index.yaml#L660-L663 - maybe it will inspire you. Or maybe just copy-paste it here. Do as you like.

Reply:
OK, maybe this will be better description:
```suggestion
    Ping method sends request that will update session values without creating new event or page view. Most common use for this method is update of session custom dimensions or custom variables.
```
Other uses for HTTP API ping (like Page Performance Metrics, prolong the session) doesn't apply here, since we have dedicated methods to send such ping requests (like automatic gathering navigation timing page performance metrics at the initiation of the tracker and heart beat setup). This API is dedicated only to allow user to force tracker to send ping request and update of custom dimension values was genesis for this feature.

Reply:
Fine by me.